app/src/main/res/values/colors.xml

```diff
@@ -10,4 +10,38 @@
     <color name="text_blue">#ff33b5e5</color>
     <color name="text_green">#ff99cc00</color>
     <color name="light_grey">#cccccc</color>
+
+    <!-- FirstStartActivity welcome wizard start -->
+    <!-- Screens background color-->
+    <color name="bg_screen1">#3395ff</color>
+    <color name="bg_screen2">#20d2bb</color>
+    <color name="bg_screen3">#c873f4</color>
+    <!-- <color name="bg_screen4">#f64c73</color> -->
```

Comment:
Is there harm in having these uncommented if there are only 3 layouts?

Reply:
Uncommenting will throw a lint warning but not be a problem. I'll reserved the color as I already know what the fourth slide will be.